README.md

```diff
@@ -2,18 +2,19 @@
 
 ![GitHub release (latest by date)](https://img.shields.io/github/v/release/gedsh/invizible?style=plastic)
 ![GitHub Releases](https://img.shields.io/github/downloads/gedsh/invizible/latest/total?color=blue&style=plastic)
-
-### [Google Play stable version](https://play.google.com/store/apps/details?id=pan.alexander.tordnscrypt.gp)
-
-### [Download the latest version from Github](https://github.com/Gedsh/InviZible/releases/latest)
+[![Translation status](https://hosted.weblate.org/widgets/invizible/-/invizible/svg-badge.svg)](https://hosted.weblate.org/engage/invizible/?utm_source=widget)
 
 ### [IzzyOnDroid F-Droid beta version](https://apt.izzysoft.de/fdroid/index/apk/pan.alexander.tordnscrypt)
 
 ### [F-Droid stable version](https://f-droid.org/packages/pan.alexander.tordnscrypt.stable/)
 
-## Android application for Internet privacy and security
+### [Google Play stable version](https://play.google.com/store/apps/details?id=pan.alexander.tordnscrypt.gp)
```

Comment:
I moved Google Play to the bottom since it is the worst option.

Reply:
99% of the project support is from Google Play. Beginners usually start with it. InviZible is intended not only for geeks, but for all users, even if they are just starting their journey to privacy and anonymity.

Reply:
F-Droid is not intended for geeks. If users could be moved to other platforms then that would be great.
Especially concerning matters "privacy and anonymity".
If people aren't asking about the specifics of Google Play, then F-Droid and Izzy isn't a worse option.
If they are asking about that, it is more reason to move it down the queue.

Reply:
Users are smart enough to make their own choices. And this is not related to the translation of the project, is it?

Reply:
Great. I am just trying to list the better options for users first. It is consistent with other projects, which is a feature of localization. Providing something users can derive meaning from and get some level of familiarity from is not irrelevant.

Reply:
Do you think users are not smart enough to make their own choices? Or maybe you think that Play Market users shouldn't use InviZible, since they don't even need a piece of privacy?

Reply:
I think the Play store is way worse for privacy, I make no observations about users.
There is a preference in mentioning the better alternatives first, and that is not the Play store.
To mention the Play-store lasts fits into the strategy of paying tribute to software that is on the side of apps like this one.
People like that, and are more likely to donate because of it.

If I am somehow wrong about it, it can be changed. So far that has happened in no projects, and I have re-ordered it in many.

Reply:
You talk about dreams. The reality is different. 99% of donations come from the Play Market. And I respect those people who need the Play Market, but don't forget to support open source projects.

Reply:
I am saying you deserve more of those.
Google prevents asking for donations in app store listings.
Google taking a 30% cut of sales is a good reason to list it last.
If you make a Liberapay account I can help you promote it and write a pitch for why people should donate.
Given the option to use all three stores and all donation platforms, people can only decide for themselves what they should donate to.

My experience is people donate more to libre software rather than closed-source software, and even more so if all opportunities to explore libre software are explored.

Doesn't that sound like a good plan? Why otherwise does the loyalty to the Play store mean it has to be listed first?

Reply:
> Google prevents asking for donations in app store listings.
Google taking a 30% cut of sales is a good reason to list it last.

Google has billions of users looking for apps like InviZible.

> If you make a Liberapay account I can help you promote it and write a pitch for why people should donate.

Thanks. I'll try.

> My experience is people donate more to libre software rather than closed-source software

Privacy and anonymity software should definitely be libre. Otherwise it makes no sense.

> Doesn't that sound like a good plan?

That sounds good. But for now, this is only a plan.

> Why otherwise does the loyalty to the Play store mean it has to be listed first?

Loyalty to users. They provide the opportunity to continue developing the app.

Reply:
Seeing how they are already users, and did pick one of the few privacy tools that can make the claim of being transparent because it is libre software, why wouldn't everyone be happy about better choices for privacy, for the same reasons, are promoted?
Try it, see if it works like it does every time, and then I will stop doing it everywhere.
In the greater scheme of things, lets focus on the strings PR, and not what I think the README should be like.
Users, and particularly more new users is what matters.

Reply:
###

Reply:
@Sarguncheema You didn't leave a comment, but I thought to add
>Google has billions of users looking for apps like InviZible.

Are they looking for it in the README of the GitHub project page?

If users are still here and so clueless as to not know the Play store and F-Droid uses tracking, the first option is the best option. Makes sense to me.